src/Redux/mission/Mission.js

```diff
@@ -0,0 +1,65 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+const SHOW_MISSION = "missions/missions/SOW_MISSION";
+const SHOW_STATUS = "missions/missions/SHOW_STATUS";
+const LEAVE_MISSIONS = "missions/missions/leaveMissions";
+
+const url = "https://api.spacexdata.com/v3/missions";
+const initialState = { missionArray: [] };
+
+export const showMission = createAsyncThunk(
+  SHOW_MISSION,
+  async (args, { dispatch }) => {
+    const response = await fetch(url);
+    const data = await response.json();
+    const missions = Object.keys(data).map((key) => {
+      const mission = data[key];
+      return {
+        id: key,
+        ...mission,
+      };
+    });
+    dispatch({
+      type: SHOW_MISSION,
+      payload: missions,
+    });
+    return missions;
```

Comment:
- [ ] Only the needed data from the missions should be dispatched. Now you are dispatching unneeded data to the store. From the requirements, only these data should be saved: `mission_id`, `mission_name`, `description`. The rest should be discarded.

  ![midd](https://user-images.githubusercontent.com/54848363/193018150-5db8cbeb-ac70-4f6f-8544-61d504db2681.jpg)

Reply:
hey @DammyShittu , thanks for the review and i did what you suggested 👍